src/controllers/message.controller.js

```diff
@@ -0,0 +1,36 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+const MESSAGE = 'message';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
+};
+
+export const postMessage = async (req, res) => {
+  const { text, author, roomId } = req.body;
+
+  if (!text || !author || !roomId) {
+    res.status(400).send({ message: 'one of nessesery parys is required' });
+  }
+
+  const message = {
+    text,
+    time: new Date().toString(),
+    author,
+    roomId,
+  };
+
+  if (!message) {
+    res.status(400).send({ message: 'message is required' });
+  }
+  await Message.create(message);
+  emitter.emit(MESSAGE, message);
+  res.status(201).send(message);
```

Comment:
In line 33, you are creating a new message and saving it to the database. However, you are not checking if the operation was successful before emitting an event and sending a response. It would be a good idea to handle possible errors in this operation.